apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.schedules/route.tsx

```diff
@@ -374,64 +385,62 @@ function SchedulesTable({
+              </div>
+              <div>
+                <div className="mb-0.5 flex items-center gap-1.5 text-sm">
+                  <div className={"flex items-center space-x-1"}>
+                    <ScheduleTypeIcon type={"IMPERATIVE"} className="text-teal-500" />
+                    <span className="font-medium">{scheduleTypeName("IMPERATIVE")}</span>
                   </div>
-                  <Paragraph variant="small" className="!text-wrap text-text-dimmed">
-                    Imperative schedules are defined here in the dashboard or by using the SDK
-                    functions to create or delete them. They can be created, updated, disabled, and
-                    deleted from the dashboard or using the SDK.
-                  </Paragraph>
                 </div>
-                <LinkButton
-                  variant="docs/small"
-                  to={docsPath("v3/tasks-scheduled")}
-                  LeadingIcon={BookOpenIcon}
-                  className="mb-1"
-                >
-                  View the docs
-                </LinkButton>
+                <Paragraph variant="small" className="!text-wrap text-text-dimmed">
+                  Imperative schedules are defined here in the dashboard or by using the SDK
+                  functions to create or delete them. They can be created, updated, disabled, and
+                  deleted from the dashboard or using the SDK.
+                </Paragraph>
               </div>
-            }
-          >
-            Type
-          </TableHeaderCell>
-          <TableHeaderCell>External ID</TableHeaderCell>
-          <TableHeaderCell>CRON</TableHeaderCell>
-          <TableHeaderCell hiddenLabel>CRON description</TableHeaderCell>
-          <TableHeaderCell>Timezone</TableHeaderCell>
-          <TableHeaderCell>Next run</TableHeaderCell>
-          <TableHeaderCell>Last run</TableHeaderCell>
-          <TableHeaderCell>Deduplication key</TableHeaderCell>
-          <TableHeaderCell>Environments</TableHeaderCell>
-          <TableHeaderCell>Enabled</TableHeaderCell>
-        </TableRow>
+              <LinkButton
+                variant="docs/small"
+                to={docsPath("v3/tasks-scheduled")}
+                LeadingIcon={BookOpenIcon}
+                className="mb-1"
+              >
+                View the docs
+              </LinkButton>
+            </div>
+          }
+        >
+          Type
+        </TableHeaderCell>
+        <TableHeaderCell>External ID</TableHeaderCell>
+        <TableHeaderCell>CRON</TableHeaderCell>
+        <TableHeaderCell hiddenLabel>CRON description</TableHeaderCell>
+        <TableHeaderCell>Timezone</TableHeaderCell>
+        <TableHeaderCell>Next run</TableHeaderCell>
+        <TableHeaderCell>Last run</TableHeaderCell>
+        <TableHeaderCell>Deduplication key</TableHeaderCell>
+        <TableHeaderCell>Environments</TableHeaderCell>
+        <TableHeaderCell>Enabled</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"